.github/workflows/autofix.yml

```
@@ -44,6 +44,9 @@
 
       - name: Install Dependencies
         run: yarn install --immutable
+        
+      - name: Set PnP Loader
+        run: echo "NODE_OPTIONS=--require ./.pnp.cjs" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:45: Double quote to prevent globbing and word splitting

(shellcheck)